devtools/mockslack/authtest.go

```diff
@@ -0,0 +1,16 @@
+package mockslack
+
+import (
+	"net/http"
+)
+
+// ServeAuthTest serves a request to the `auth.test` API call.
+//
+// https://slack.com/api/auth.test
+func (s *Server) ServeAuthTest(w http.ResponseWriter, req *http.Request) {
+	var respData struct {
+		TeamID string `json:"team_id"`
+	}
+	respData.TeamID = genID(idChars, 9)
```

Comment:
This should be persistent, maybe add `TeamID` to `state` and add a teamID gen method to `gen` that begins with `T`